gui/default/assets/lang/lang-en.json

```diff
@@ -258,12 +268,14 @@
     "More than a year ago": "More than a year ago",
     "Move to top of queue": "Move to top of queue",
     "Multi level wildcard (matches multiple directory levels)": "Multi level wildcard (matches multiple directory levels)",
+    "Name": "Name",
```

Comment:
Might need to disambiguate here (use a dotted namespace key), because there is no context and translations may vary by what is given a name.

Reply:
Fixed in 4d5b84af835b56f23281a96c3d756ad0d04b846b.